internal/clients/new_instance_types_test.go

```diff
@@ -0,0 +1,74 @@
+package clients_test
```

Comment:
Can you rename `embed.go` to something reasonable and this file to `something_reasonable_test.go`?

The Idiomatic Go pattern is always keep unit tests named as their source files:

* blah.go
* blah_test.go

Reply:
Yes! It bothered me as well 😀